nginx.tmpl

```diff
@@ -144,12 +144,11 @@
     ssl_protocols TLSv1.3;
         {{- /*
              * nginx currently lacks ability to choose ciphers in TLS 1.3 in
-             * configuration; see https://trac.nginx.org/nginx/ticket/1529.  A
-             * possible workaround can be modify /etc/ssl/openssl.cnf to change
-             * it globally (see
-             * https://trac.nginx.org/nginx/ticket/1529#comment:12).  Explicitly
-             * set ngnix default value in order to allow single servers to
-             * override the global http value.
+             * configuration, see https://trac.nginx.org/nginx/ticket/1529.
```

Comment:
The linked bug is closed as fixed. Is this comment still valid?

Reply:
I asked myself the same question a few months ago, I did not have time to properly test yet.